src/master-playlist-controller.js

```diff
@@ -258,6 +259,12 @@ export class MasterPlaylistController extends videojs.EventTarget {
 
     this.setupSegmentLoaderListeners_();
 
+    if (this.experimentalBufferBasedABR) {
+      this.startABRTimer_();
+      this.tech_.on('pause', () => this.stopABRTimer_());
```

Comment:
I wonder if we should handle seeks that cause a pause differently here. Should they be ignored so that we can switch playlists during a seek? Maybe that already happens?

Reply:
Was thinking about this a bit more and perhaps we just want to call `this.checkABR_` on `seeking` so that we can change playlist during a seek rather than afterwards?

Reply:
If we're just seeking by clicking, the pause is so small as to not matter. If we're scrubbing, we'll get a whole bunch of seeking events, so, we'd want to debounce it or something.
Maybe `stopABRTimer_` should check whether scrubbing is set? So, we only stop it if we're paused for not scrubbing.

Reply:
Updated the stop handler to check `tech.scrubbing()` if it exists, which is being added here https://github.com/videojs/video.js/pull/6920